scalableminds/webknossos
# Fix cachedMaxNodeId after nml import

The cachedMaxNodeId was not properly updated in the addTrees reducer that is used during nml import. Therefore, nodes that were added later had duplicate ids :/

### Steps to test:
- Import an NML using the "Import NML" functionality. Add a new tree and some nodes afterwards. The nodes should be visible, mergable, ...

------
